src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  /* height: 60px; */
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  position: relative;
+}
+
+.nav__list {
+  text-decoration: none;
+  margin-left: 20px;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  font-style: normal;
+  font-family: "Roboto", sans-serif;
```

Comment:
Set font styles to `html` tag

Reply:
font-size: 12px;
font-weight: 500;

Is this too?

Reply:
Yep